docs/sphinx/source/whatsnew/v0.11.2.rst

```diff
@@ -47,6 +47,9 @@ Documentation
   page, using the sphinx glossary directive. (:issue:`1421`, :pull:`2234`)
 * Explained how to write docstrings for new functions in :ref:`example-docstring`
   (:discussion:`2081`, :pull:`2254`)
+* Clarify mounting cases for parameters for :py:func:`~pvlib.temperature.sapm_module`
+  :py:func:`~pvlib.temperature.sapm_cell` and :py:func:`~pvlib.temperature.pvsyst_cell`.
+   (:issue:`1323`, :pull:`2293`)
```

Comment:
The HTML rendering is a bit messed up here, but let's just fix it along with the other touch ups in the release finalization PR